### PR TITLE
[Gateway]  new setting ignore_cname_category_matches

### PR DESCRIPTION
**Pull request by rexscaria**
Change log update

### CODE REVIEW

File: data/changelogs/gateway.yaml [hunk 1]
```diff
@@ -5,6 +5,10 @@ productLink: "/cloudflare-one/policies/gateway/"
 productArea: Cloudflare One
 productAreaLink: /cloudflare-one/changelog/
 entries:
+- publish_date: '2024-06-25'
+  title: New Gateway DNS policy settings for ignoring CNAME category matches
+  description: |-
+    Gateway now offers a way to selectively ignore CNAME domain categories in DNS policies through `ignore_cname_category_matches`. [Doc](https://developers.cloudflare.com/api/operations/zero-trust-gateway-rules-create-zero-trust-gateway-rule)
```
Comment (cdisher) [inline data/changelogs/gateway.yaml:11]: Thanks for doing this Rex! Can we change "a way" to "the ability"?

Gateway now offers the ability to selectively ignore CNAME domain categories in DNS policies through `ignore_cname_category_matches` in the [API](https://developers.cloudflare.com/api/operations/zero-trust-gateway-rules-create-zero-trust-gateway-rule).

Comment (cdisher) [reply data/changelogs/gateway.yaml:11]: Or can do: Gateway now offers the ability to selectively ignore CNAME domain categories in DNS policies. [API Documentation](https://developers.cloudflare.com/api/operations/zero-trust-gateway-rules-create-zero-trust-gateway-rule)

Comment (cdisher) [inline data/changelogs/gateway.yaml]:
```suggestion
    Gateway now offers the ability to selectively ignore CNAME domain categories in DNS policies through `ignore_cname_category_matches`. [API Documentation](https://developers.cloudflare.com/api/operations/zero-trust-gateway-rules-create-zero-trust-gateway-rule)
```

File: data/changelogs/gateway.yaml [hunk 2]
```diff
@@ -5,6 +5,10 @@ productLink: "/cloudflare-one/policies/gateway/"
 productArea: Cloudflare One
 productAreaLink: /cloudflare-one/changelog/
 entries:
+- publish_date: '2024-06-25'
```
Comment (maxvp) [inline data/changelogs/gateway.yaml:8]:
```suggestion
- publish_date: '2024-06-30'
```

File: data/changelogs/gateway.yaml (hunk 1 above)
Comment (maxvp) [reply data/changelogs/gateway.yaml]:
```suggestion
    Gateway now offers the ability to selectively ignore CNAME domain categories for DNS policies via the [`ignore_cname_category_matches` setting](/api/operations/zero-trust-gateway-rules-create-zero-trust-gateway-rule).
```

Comment (rexscaria) [reply data/changelogs/gateway.yaml]: Applied

**Comment by rexscaria** [comment]
Can we merge this or lemme know if we need more suggestions @maxvp

**Review by maxvp — APPROVED** [review]

**Comment by maxvp** [comment]
Approved and merged!

**Pull request merged**